sites/platform/src/create-apps/app-reference.md

```diff
@@ -193,50 +193,161 @@ For more information, see how to [manage resources](/manage-resources.md).
-This name is used in the `PLATFORM_RELATIONSHIPS` environment variable,
-which gives you credentials for accessing the service.
+This name is then used in the [`PLATFORM_RELATIONSHIPS` environment variable](/development/variables/use-variables.md#use-provided-variables),
+which provides the details of the relationship including, for example, the credentials to access the service.
 
-The relationship is specified in the form `service_name:endpoint_name`.
-The `service_name` is the name of the service given in the [services configuration](../add-services/_index.md)
-or the name of another application in the same project specified as the `name` in that app's configration.
+{{% version/specific %}}
+<!-- Platform.sh -->
+To define a relationship, use one of the following syntaxes in your `{{% vendor/configfile "app" %}}` file.
 
-The `endpoint_name` is the exposed functionality of the service to use.
-For most services, the endpoint is the same as the service type.
-For some services (such as [MariaDB](../add-services/mysql/_index.md#multiple-databases) and [Solr](../add-services/solr.md#solr-6-and-later)),
-you can define additional explicit endpoints for multiple databases and cores in the [service's configuration](../add-services/_index.md).
+<--->
+<!-- Upsun -->
+To define a relationship, use one of the following syntaxes in your `{{% vendor/configfile "app" %}}` file,
+under the `applications` top-level key.
+{{% /version/specific %}}
 
-The following example shows a single MySQL service named `mysqldb` offering two databases,
-a Redis cache service named `rediscache`, and an Elasticsearch service named `searchserver`.
+{{< codetabs >}}
+
++++
+title=Standard syntax
++++
+
+```yaml {configFile="app"}
+relationships:
+  database: # The name of the relationship. 
+    service: {{% variable "SERVICE_NAME" %}} 
+    endpoint: {{% variable "ENDPOINT_NAME" %}}
+```
+
+- `SERVICE_NAME`: The name of the service as defined in its [configuration](/add-services/_index.md).
+- `ENDPOINT_NAME`: The endpoint to connect to the service.
+  For most services, the endpoint is the same as the service type.
+  For some services (such as [MariaDB](../add-services/mysql/_index.md#multiple-databases) and [Solr](../add-services/solr.md#solr-6-and-later)),
+  you can define additional explicit endpoints for multiple databases and cores in the [service's configuration](../add-services/_index.md).</br>
+  Every service container has a default endpoint that is used if no endpoint is defined.
+
+For example, to define a relationship named `database` that connects your app to a service called `mariadb` through the `mysql` endpoint,
+use the following configuration:
+
+```yaml {configFile="app"}
+relationships:
+  database: # The name of the relationship. 
+    service: mariadb
+    endpoint: mysql
+```
+
+{{< note >}}
+To connect your app to another app in your project, replace the `SERVICE_NAME` by the name of that other app,
```

Comment:
Given

> Every service container has a default endpoint that is used if no endpoint is defined.

Is it smart enough to know that `endpoint: http` is the default to be applied?

If so, maybe we can include

```yaml {configFile="app"}
relationships:
    api:
        service: backend
```

And _actually_ is it ever necessary that a user knows about `http` in the background if this is the way we're handling it now?

Reply:
My assumption was bad actually, app containers don't have a default `http` endpoint. Updated accordingly.